### ISSUE TITLE
FDMexico

### ISSUE THREAD

**Issue by joskua** [Fiserv/Support#231]
**Region** (optional)
Mexico

**Is your feature request related to a problem? Please describe.**
We would like to migrate our existing documentation at [https://docs.firstdata.com/org/FDMexico](https://docs.firstdata.com/org/FDMexico) to dev Studio. What should be the next steps?

(didn´t find the way to add the New Tenant Tag)

**Comment by sfiserv** [comment]
Created a tracking ticket in order to add the Tenant on the Devstudio UI. 
https://firstdatateam.atlassian.net/browse/PORT-5466

**Comment by joskua** [comment]
Hello team I don't know  If I should but I do not have permission to see the ticket. 
![image](https://user-images.githubusercontent.com/117851284/202310056-98cab2d9-ab73-45e4-bd99-92d8bb0ca48b.png)


> Created a tracking ticket in order to add the Tenant on the Devstudio UI. https://firstdatateam.atlassian.net/browse/PORT-5466

**Comment by TaniaSarkar89** [comment]
Hi @joskua the JIRA ticket is for internal Developer Studio team only. 

We would update the status here as and when this GitHub ticket is addressed.

Thanks!

**Comment by sfiserv** [comment]
@joskua to get started with your product, Devstudio team will create a repo in github. fro that please provide us the Product name which will be displayed on the Devstudio UI. Along with it please provide us the solution ( Merchants or FInancial Institutions) under which the product should be displayed. If you need more information or a comparative study for this you can visit our url https://developer.fiserv.com/.

**Comment by joskua** [comment]
On our [previous dev portal](https://docs.firstdata.com/org/FDMexico) we managed the following products for Merchants:

Tarjeta No Presente (Card Not Present):

- API REST de Pagos
- API SOAP de Pagos
- Connect
- Plugins
  - Magento
  - Woocommerce
  - Prestashop

Tarjeta Presente (Card Present):

- Pinpads
- FDGo MPoS

**Comment by joskua** [comment]
On our [previous dev portal](https://docs.firstdata.com/org/FDMexico) we managed the following products for Merchants:

Tarjeta No Presente (Card Not Present):

- API REST de Pagos
- API SOAP de Pagos
- Connect
- Plugins
  - Magento
  - Woocommerce
  - Prestashop

Tarjeta Presente (Card Present):

- Pinpads
- FDGo MPoS

**Comment by TaniaSarkar89** [comment]
Hello @joskua could you please fill up all the information and submit this [onboarding form](https://github.com/Fiserv/Support/issues/new?assignees=sfiserv&labels=Onboard%20New%20Tenant&template=onboard_new_tenant.md)
We would setup the tenant in our lower information based on the details you provide.

**Comment by TaniaSarkar89** [comment]
Hi @joskua were you able to submit this [onboarding form](https://github.com/Fiserv/Support/issues/new?assignees=sfiserv&labels=Onboard%20New%20Tenant&template=onboard_new_tenant.md) ?

**Comment by joskua** [comment]
Hello @TaniaSarkar89  we are on the process oh signing up the whole team. Should we make an onboarding form for every product from which we have documentation?

**Comment by TaniaSarkar89** [comment]
Yes @joskua you should submit individual onboarding forms for individual products/tenants.

**Comment by TaniaSarkar89** [comment]
The tenant Onboarding will be done once the Onboarding form is submitted. Closing this ticket for now.
